packages/odie-client/src/constants.ts

```diff
@@ -66,6 +66,61 @@ export const ODIE_TRANSFER_MESSAGE = (
 	},
 } );
 
+export const ODIE_TRANSFER_MESSAGE_NEW = (
```

Comment:
@agrullon95 can we do something like we did for `getOdieInitialMessageConstant`? Create a function to return the right constant?